src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -27,15 +27,22 @@ export class BaselinesTable extends Component {
         super(props);
 
         this.state = {
-            sortBy: {
-                index: 1,
-                direction: 'asc'
+            params: {
+                sortBy: {
+                    index: 1,
+                    direction: 'asc'
+                },
+                search: undefined,
+                orderBy: 'display_name',
+                orderHow: 'ASC',
+                page: 1,
+                perPage: 20
             },
-            search: undefined,
-            orderBy: 'display_name',
-            orderHow: 'ASC',
-            page: 1,
-            perPage: 20
+            bulkSelectType: '',
+            errorMessage: [ 'The list of baselines cannot be displayed at this time. Please retry and if',
```

Comment:
Do we need to keep error message always in the table component? Maybe it is better to keep error flag/API response status in the params, message in some constants/messages file. Then map the error with corresponding message when an error happened.

Reply:
Do you mean to simply save the error message as a constant in another constants file and then import the error message when needed?

I'm fine to do that, but only this component uses the error message, which is why I left it here.

I don't care either way though.

Reply:
In that case, it makes sense